### ISSUE TITLE
Loading Zone

### ISSUE THREAD

**Issue by jlpoolen** [curbmap/Open-Street-Parking-Restriction-Specification#6]
Are loading zones "yes" or "no" for parking?

**Comment by eselkin** [comment]
Good question. Probably, yes parking, but then there has to be some area for time limit. It probably has to be somewhere in the extensions area. Or maybe we should have a time limit section in the main body of the spec.

**Comment by jlpoolen** [comment]
Perhaps your data model should have a CURB_TYPES table that contains the variety of curbs activities: a) parking, 2) loading, 3) no stopping, 4) whatever else comes to your attention

another thought: relationship of curb to vehicle:
1) standard curb where car parks parallel
2) standard curb where car parks perpendicular
3) standard curb where car parks at angle, e.g. 45 degrees

**Comment by eselkin** [comment]
I agree about the CURB_TYPES property and about the parking type, we probably also have to have an option if a city wants to use this for cataloging "off-street" public parking.
